index.html

```diff
@@ -14,7 +14,7 @@
   <body>
     <header>
       <a href="#"><img src="My Logo.svg" alt="Logo svg" /></a>
-      <nav>
+      <nav class="menu-button">
         <img src="Union.png" alt="Menu Button" />
       </nav>
     </header>
```

Comment:
There is still an empty/blank space to the right of the header and the about section. Kindly check and rectify that. You are almost there. 

![Screenshot from 2022-09-24 02-35-29](https://user-images.githubusercontent.com/65233860/192076564-0c52a094-77de-4f39-9ab7-3d374daa6ad1.png)

Reply:
Hi, I tried to remove that space by using a flexbox container and centering the content, but it isn't working. If that's what you are talking about.

Reply:
well done @sebalp100 for implementing this change. Sorry I could not see your message earlier. Next time you can tag me or send me a message on slack. Congrats again. :100: